workspaces/liferay-sample-workspace/client-extensions/liferay-sample-etc-node/util/log.js

```diff
@@ -0,0 +1,14 @@
+'use strict';
+
+import {createLogger, format, transports} from 'winston';
```

Comment:
Rather than rely on an external package, can we just use `console.log`? The more dependencies we can remove, the better, as we aim for direct and simple samples.